### ISSUE TITLE
get_neighborhood.py TypeError: 'int' object is not iterable

### ISSUE THREAD

**Issue by samlipworth** [gtonkinhill/panaroo#54]
Hi - getting this error when running get_neighborhood.py:

```
Traceback (most recent call last):
  File "getneighbours.py", line 101, in <module>
    main()
  File "getneighbours.py", line 76, in main
    G.nodes[n]['members'] = set(G.nodes[n]['members'])
TypeError: 'int' object is not iterable
```

I've cloned the latest repository to make sure that wasn't the problem. Do you know what is causing this?

**Comment by gtonkinhill** [comment]
This was caused by gene clusters present in only one genome. I have hopefully fixed this in the latest release on GitHub

**Comment by samlipworth** [comment]
Many thanks for looking at this. It now runs successfully, but the output is not like that in the tutorial, I get a list eg:

gene,dist
group_80,1
glpR_3,1
group_9736,2
group_7980,2
lacY_2,3
group_9602,3
group_7864,3
sasA_2,3
nqrC,3
group_1009,3
group_6376,3
adeQ__adeQ_2,4
group_922,4
group_10322,4
resA_1,4
group_6561,4
walR,4
group_2376,4
srpC_2,4
arcA_2,4
group_223,4
group_319,4
group_3509,4

It's not imediately clear to me what the paths through this are whereas it is in the tutorial and i also cannot tell which genomes support which path. Is it possible to recreate the output in the tutorial?

**Comment by gtonkinhill** [comment]
Hi, apologies for being a bit slow to get back to this. 

I had not realised we had documented this script and I agree the original output format was more useful. I have just pushed an update that is more similar to the original but hopefully a little easier to parse. Columns are now tab separated with genes within paths being separated by commas. 

I've updated the [documentation](https://gtonkinhill.github.io/panaroo/#/post/gene_neighbourhood) which should hopefully match up with the updated script.
